components/Overlay/PositionMarketOverlay/index.tsx

```diff
@@ -0,0 +1,134 @@
+                    option={currentMarket}
+                    keyMap={marketKeyMap}
+                    defaultValue="Select Market"
+                />
+            ) : null}
+
+            {currentMarket === -1 ? (
+                <MarketPreviewContainer>
+                    <InfoCol>
+                        <div className="title">Market</div>
+                        <div className="row">
+                            <SLogo ticker={tracers[Object.keys(tracers)[0]]?.baseTicker} />{' '}
+                            {tracers[Object.keys(tracers)[0]]?.marketId}
+                        </div>
+                    </InfoCol>
+                    <InfoCol>
+                        <div className="title">Last Price</div>
+                        <div className="row">
+                            {toApproxCurrency(tracers[Object.keys(tracers)[0]]?.getOraclePrice())}
+                        </div>
+                    </InfoCol>
+                    <InfoCol>
+                        <div className="title">24h</div>
+                        <div className="row">
+                            <MarketChange amount={tracers[Object.keys(tracers)[0]]?.get24HourChange()} />
+                        </div>
+                    </InfoCol>
+                    <InfoCol>
+                        <div className="title">Max Leverage</div>
+                        <div className="row">{`${tracers[Object.keys(tracers)[0]]?.getMaxLeverage()}x`}</div>
+                    </InfoCol>
+                </MarketPreviewContainer>
+            ) : (
+                <MarketPreviewContainer>
+                    <InfoCol>
+                        <div className="title">Market</div>
+                        <div className="row">
+                            <SLogo ticker={tracers[Object.keys(tracers)[currentMarket]].baseTicker} />{' '}
+                            {tracers[Object.keys(tracers)[currentMarket]].marketId}
+                        </div>
+                    </InfoCol>
+                    <InfoCol>
+                        <div className="title">Last Price</div>
+                        <div className="row">
+                            {toApproxCurrency(tracers[Object.keys(tracers)[currentMarket]].getOraclePrice())}
+                        </div>
+                    </InfoCol>
+                    <InfoCol>
+                        <div className="title">24h</div>
+                        <div className="row">
+                            <MarketChange amount={tracers[Object.keys(tracers)[currentMarket]].get24HourChange()} />
+                        </div>
+                    </InfoCol>
+                    <InfoCol>
+                        <div className="title">Max Leverage</div>
+                        <div className="row">{`${tracers[Object.keys(tracers)[currentMarket]].getMaxLeverage()}x`}</div>
+                    </InfoCol>
+                </MarketPreviewContainer>
+            )}
+        </StyledOverlay>
```

Comment:
What case is this handling for. If currentMarket === -1 (ie no market is selected) then display the first market why you could just set currentMarket = 0 as a default couldnt you?

Reply:
0 is an index in the fetched factory tracers object so need to avoid using currentMarket 0 here as it will interfere

Reply:
and we don't want users to select 'Select Market' once they have selected a market